app/actions/add-new-goal.js

```diff
@@ -1,10 +1,30 @@
-import { INPUT_GOAL } from './../action_types.js';
+import * as types from './../action_types.js';
```

Comment:
I think you save characters here with
```js
import { INPUT_GOAL } from '../action_types.js' 
```

It is in the case when you will just pass on the object the import * as gives you without trying to get its properties.